integration/travis.sh

```diff
@@ -12,4 +12,19 @@ sudo mkdir -p /etc/servicebuilder.d /var/service-stage /var/service
 
 sudo cp $GOPATH/bin/p2-exec /usr/local/bin
 
+# make ssl certs
+subj="
+C=US
+ST=CA
+O=SQ
+localityName=SF
+commonName=$HOSTNAME
+organizationalUnitName=Vel
+emailAddress=doesntmatter@something.edu
+"
+CERTPATH=/var/tmp/certs
+mkdir -p $CERTPATH
+openssl req -x509 -newkey rsa:2048 -keyout $CERTPATH/key.pem -out $CERTPATH/cert.pem -nodes -days 300 -subj "$(echo -n "$subj" | tr "\n" "/")"
```

Comment:
I'm surprised this works in Travis!  Last time I tried submitting code that created asymmetric keys in a test, it caused the test to time out.

In developing this test, have you seen _any_ of your test cases fail due to runtime problems?